doc/user/content/manage/dbt/development-workflows.md

```diff
@@ -360,6 +360,17 @@ These environments are later swapped transparently.
     deployment, which transparently handles running that subset of models
     against the deployment environment.
 
+    If you encounter an error like `String 'deploy:' is not valid YAML`, you
```

Comment:
What do you think about putting this in a callout block? 

Otherwise +1

Reply:
Good idea, just added it:
<img width="817" alt="image" src="https://github.com/user-attachments/assets/762b432a-bc09-40b7-9607-5e40f90cdb85" />